fineract-investor/src/main/resources/db/changelog/tenant/module/investor/parts/0007_add_external_asset_owner_transfer_details.xml

```diff
@@ -0,0 +1,65 @@
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <!-- Create the m_external_asset_owner_transfer_balance table -->
+    <changeSet author="fineract" id="1" context="postgresql">
+        <createTable tableName="m_external_asset_owner_transfer_details">
+
+            <column autoIncrement="true" name="id" type="BIGINT" remarks="Internal ID">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+
+            <column name="asset_owner_transfer_id" type="BIGINT" remarks="Id of asset owner transfer">
+                <constraints nullable="false" unique="true"
+                             uniqueConstraintName="UQ_external_asset_owner_transfer_id"/>
+            </column>
+
+            <column name="total_outstanding_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="principal_outstanding_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="interest_outstanding_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="fee_charges_outstanding_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="penalty_charges_outstanding_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="total_overpaid_derived" type="DECIMAL(19, 6)">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="created_by" type="BIGINT" />
+            <column name="created_on_utc" type="TIMESTAMP WITH TIME ZONE" />
+            <column name="last_modified_by" type="BIGINT" />
+            <column name="last_modified_on_utc" type="TIMESTAMP WITH TIME ZONE" />
+        </createTable>
+    </changeSet>
```

Comment:
please dont forget the changeset for mysql/mariadb

Reply:
added